File: snapshots/input/syntax/Main/Enums.cs
using System.Diagnostics.CodeAnalysis;

namespace Main;

[SuppressMessage("ReSharper", "all")]
public class Enums
{
    enum EnumWithIntValues
    {
        Ten = 10,
        Twenty = 20
    }

    enum EnumWithByteValues
    {
        Five = 0x05,
        Fifteen = 0x0F
    }
}

File: snapshots/input/syntax/VBMain/Enums.vb
Imports System.Diagnostics.CodeAnalysis

Namespace VBMain
    <SuppressMessage("ReSharper", "all")>
    Public Class Enums
        Enum EnumWithIntValues
            Ten = 10
            Twenty = 20
        End Enum

        Enum EnumWithByteValues
            Five = &H5
            Fifteen = &HF
        End Enum
    End Class
End Namespace